hack/util.sh

```diff
@@ -48,6 +35,44 @@ KARMADA_GREETING='
 ------------------------------------------------------------------------------------------------------
 '
 
+function util::karmada_target_source() {
+    case $1 in
+     karmada-aggregated-apiserver)
+            echo "cmd/aggregated-apiserver"
+          ;;
```

Comment:
```suggestion
            ;;
```

Reply:
How do you think about it
```sh
KARMADA_TARGET_SOURCE=(
  karmada-aggregated-apiserver=cmd/aggregated-apiserver
  karmada-controller-manager=cmd/controller-manager
  karmada-scheduler=cmd/scheduler
  karmada-descheduler=cmd/descheduler
  karmadactl=cmd/karmadactl
  kubectl-karmada=cmd/kubectl-karmada
  karmada-webhook=cmd/webhook
  karmada-agent=cmd/agent
  karmada-scheduler-estimator=cmd/scheduler-estimator
  karmada-interpreter-webhook-example=examples/customresourceinterpreter/webhook
)

function util::get_target_source() {
  local target=$1
  for s in "${KARMADA_TARGET_SOURCE[@]}"; do
    if [[ "$s" == ${target}=* ]]; then
      echo "${s##${target}=}"
      return
    fi
  done
}
```

Reply:
thank you very much ,let me benefit a lot